matplotlib_scalebar/scalebar.py

```diff
@@ -111,11 +112,17 @@ def _validate_legend_loc(loc):
     }
 )
 
+
+if Version(matplotlib.__version__ ) >= Version('3.5.dev'):
```

Comment:
I'm just wondering whether this would be simpler to avoid the dependency to the `packaging` module:

```python
_all_deprecated = getattr(matplotlib, "_all_deprecated", {})
```

Reply:
Yes, this is better! :)